test/controllers/record_controller_test.rb

```diff
@@ -9,7 +9,8 @@ class RecordControllerTest < ActionDispatch::IntegrationTest
 
   test 'full record path with an id does not return an error' do
     VCR.use_cassette('timdex record sample',
-                     allow_playback_repeats: true) do
+                     allow_playback_repeats: true,
+                     match_requests_on: %i[method uri body]) do
```

Comment:
Great call to use this feature to match on the body as well for these now that we've moved to GraphQL. I had a hard time wrapping my head around it this morning, but I'm 100% on board now.